benchmark/shear_bands/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.8)
 
-FIND_PACKAGE(Aspect REQUIRED HINTS ${ASPECT_DIR} ../ ../../ $ENV{ASPECT_DIR})
+FIND_PACKAGE(Aspect REQUIRED HINTS ${Aspect_DIR} ../ ../../ $ENV{ASPECT_DIR})
```

Comment:
Could we just use both `${Aspect_DIR}` and `${ASPECT_DIR}` as hints here? Same everywhere else.